gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/Cluster.java

```diff
@@ -464,6 +465,14 @@ public boolean isUserAgentOnConnectEnabled() {
         return manager.isUserAgentOnConnectEnabled();
     }
 
+    /**
+     * Checks if cluster is configured to send bulked results
+     * in the web socket handshake
```

Comment:
This is an inaccurate comment.

Reply:
Oh ops, that was a bad copy paste and partial edit